pkg/config/tls.go

```diff
@@ -33,6 +33,16 @@ type TLS struct {
 	RootCA      string `name:"root-ca" description:"Location of TLS root CA certificate (optional)"`
 	Certificate string `name:"certificate" description:"Location of TLS certificate"`
 	Key         string `name:"key" description:"Location of TLS private key"`
+	ACME        ACME   `name:"acme"`
+}
+
+// ACME represents ACME configuration.
+type ACME struct {
+	Enable   bool     `name:"enable" description:"Enable automatic Let's Encrypt certificates"`
+	Endpoint string   `name:"endpoint" description:"ACME endpoint (optional)"`
```

Comment:
This implies we can use also something else than Let's Encrypt, correct?

If so, should we call "Let's Encrypt" the default and use that in examples?

Reply:
Yeah, but in most cases this will just be used to switch to Let's Encrypt's staging server.

Will change

Reply:
OK, that is because that's the default of the package, right?

Reply:
Added a commit to explicitly set it, so now `--help` shows

```
--tls.acme.endpoint string ACME endpoint (default "https://acme-v01.api.letsencrypt.org/directory")
```